content/_data/changelogs/weekly.yml

```diff
@@ -1534,6 +1534,34 @@
       pull: 3122
       message: >
         Developer: Implement default methods in <code>TaskListener</code> and <code>BuildListener</code> interfaces so they don't have to be implemented in subclasses.
+- version: "2.92"
+  date: 2017-11-26
+  changes:
+    - type: rfe # https://github.com/jenkinsci/jenkins/pull/3154#issuecomment-345983962
+      pull: 3154
+      issue: 48116
+      message: >
+        Revert internal change that broke assumption in <code>ruby-runtime</code> in 2.91, impacting plugins based on it.
+    - type: rfe
+      pull: 3143
+      message: >
+        Improve UI performance with long list of running builds by caching the estimated duration.
+    - type: rfe
+      pull: 3155
+      message: >
+        Cache permission names, allowing Jenkins to recover faster after "stop-the-world" Java GC pauses.
+    - type: bug
+      pull: 3156
+      issue: 48157
+      message: >
+        Prevent potential <code>NullPointerException</code> when migrating the default "All View" name for a "My Views" user property.
+    - type: rfe
+      pull: 3149
+      message: >
+        Developer: Add <code>AccessControlled#hasPermission(Authentication, Permission)</code> for convenience.
```

Comment:
Add the `<code>AccessControlled#hasPermission(Authentication, Permission)</code>` convenience method ?